### PR TITLE
Adjust the symbols list

### LINKED ISSUE #6101: Symbol list is too small in NVDA Symbol / Punctuation dialog.
Running Windows 10, NVDA 2016.2.1, latest next or back to 2015.4 (earliest build I have handy) all work the same.

On the Symbol / Punctuation dialog the symbol list takes up about 60% of the width of the dialog yet only displays the "symbol" and "replacement" fields.  The "level" field is partially displayed, but the "Preserve" field is not displayed at all.  Neither the symbol or preserve field are wide enough as it is to display their contents visually.

Proposed solution:
Put the "Symbol" label above the list and have the list itself take up the full width (minus a small margin) of the dialog.

### PR DESCRIPTION
**Pull request by feerrenrut**
Fixes #6101
Extends PR #6178 

Put the 'symbol' label above the list and have the list itself take up the full width of the dialog. Columns are spaced automatically, with one nominated to take up any extra available room.

The first column was selected to expand more, as the information in the other columns can be viewed in the "change symbol" group.

### CODE REVIEW

**Comment by feerrenrut** [comment]
@jcsteh Would you mind reviewing this?

File: source/gui/nvdaControls.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: UTF-8 -*-
+#settingsDialogs.py
+#A part of NonVisual Desktop Access (NVDA)
+#Copyright (C) 2006-2016 NV Access Limited
```
Comment (jcsteh) [inline source/gui/nvdaControls.py]: Please change file name above and copyright year to just 2016.

File: source/gui/nvdaControls.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: UTF-8 -*-
+#settingsDialogs.py
+#A part of NonVisual Desktop Access (NVDA)
+#Copyright (C) 2006-2016 NV Access Limited
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
```
Comment (jcsteh) [inline source/gui/nvdaControls.py]: Blank line below this please.

File: source/gui/nvdaControls.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+import wx
+from wx.lib.mixins import listctrl as listmix
+
+class AutoWidthColumnListCtrl(wx.ListCtrl, listmix.ListCtrlAutoWidthMixin):
+	"""
+	A list control that allows you to specify a column to resize to take up the remaining width of a wx.ListCtrl
+	"""
+	def __init__(self, parent, ID, autoSizeColumnIndex="LAST", pos=wx.DefaultPosition, size=wx.DefaultSize, style=0):
```
Comment (jcsteh) [inline source/gui/nvdaControls.py]: wx controls usually take `id` as an optional argument defaulting to `wx.ID_ANY`. So, please change `ID` to `id=wx.ID_ANY`.

**Comment by feerrenrut** [comment]
Accidentally referenced the wrong PR in the commit message. See: commit e533369ea7f2a8801cec5ce2f94dcca6855ae8c0 for incubating commit

**Comment by feerrenrut** [comment]
This has been merged into master with commit: 87ed4d17feee64b4ea021a125baed1dbba18a187

**Pull request closed without merging**